fastlane/lib/fastlane/actions/sync_code_signing.rb

```diff
@@ -10,7 +10,6 @@ def self.run(params)
         require 'match'
 
         params.load_configuration_file("Matchfile")
-        params[:api_key] ||= Actions.lane_context[SharedValues::APP_STORE_CONNECT_API_KEY]
```

Comment:
Not needed anymore because `Match` will look for existing `Spaceship::ConnectAPI.client` now and use that